source/agora/common/crypto/Schnorr.d

```diff
@@ -269,3 +308,16 @@ nothrow @nogc @safe unittest
     assert(verify(kp2.V, sig2, secret));
     assert(!verify(kp2.V, sig1, secret));
 }
+
+unittest
```

Comment:
Is this doing anything the other tests are not testing already? If it is, it would be good to document it - otherwise it would be unnecessary to duplicate tests.

Reply:
I updated an existing test in Key to use the fromScalar construction so removed this one.